scrapers/XConfessions.yml

```diff
@@ -62,4 +89,55 @@ xPathScrapers:
       FrontImage:
         selector: //div[@class="w-1/3 hidden tablet:block"]//@data-srcset
         postProcess: *pp
-# Last Updated May 31, 2021
+
+  performerScraper:
+    common:
+      $data: //main
+      $script: //script[contains(.,"window.__NUXT")]/text()
+    performer:
+      Name: $data//h1[contains(@class, "leading-none")]/span/../text()
+      URL: //link[@data-n-head='ssr' and @rel='canonical']/@href
+      Birthdate: 
+        selector: $data//span[text()="Birth"]/following-sibling::div
+        postProcess:
+          - parseDate: 2006
+      Country: 
+        selector: $data//span[text()="Birthplace"]/following-sibling::div
+        postProcess:    
+          - replace:
+              - regex: ^(.*?),\s
+                with: ""
+      Details: $data//div[contains(@class, "description-block")]/p
+      Images:
+        selector: $data//img[contains(@class, "image") and contains(@class, "absolute") and contains(@class, "w-full")]/@srcset
+        postProcess:
+          - replace:
+              - regex: ^([^?]+)\?.*
+                with: $1
```

Comment:
copy paste error ? the image is already present below

Reply:
Oof
That was just for my fork/PR. forget to remove for the PR
Removed.